test-data/unit/lib-stub/builtins.pyi

```diff
@@ -20,8 +20,13 @@ class bytes: pass
 class function: pass
 class ellipsis: pass
 
-from typing import Generic, Sequence, TypeVar
+from typing import Generic, Sequence, TypeVar, Mapping
+
 _T = TypeVar('_T')
 class list(Generic[_T], Sequence[_T]): pass
 
+_KT = TypeVar('_KT')
+_VT = TypeVar('_VT')
+class dict(Mapping[_KT, _VT]): pass
```

Comment:
This is a problem -- sorry, didn't notice this earlier. As mentioned at the top of the file, we don't want anything to be added here. Also the other test stubs updates are pretty invasive. I can see a few options:

1. If `dict` is not defined, skip the new code path.
2. If `dict` is not defined, use `Any` as the type (this can be a bit confusing, but it's probably okay).

Reply:
OK, that makes sense, I'll have a look.

Reply:
@JukkaL how about this? Now errors will be suppressed if dict is not defined. I've also removed dict from most fixtures, but kept it where the corresponding tests needed to be updated.

Reply:
I've removed the dict stub from all fixtures. The new code is still tested though, via the new unit test that relies on `dict.pyi` and checks the type of `__annotations__` . I'll resolve this for now, please let me know if you want me to change it further.